storage/src/block/mod.rs

```diff
@@ -408,6 +408,10 @@ impl BlockStorage {
         self.dag_sync_block_storage.remove(block_id)
```

Comment:
I see you have implemented this function. I don't know where do you want to call  this function. It likes garbarge collection produce. If your don't call this function, the dag_sync_block storage grows infinitely.